PSL_catalog.json

```diff
@@ -0,0 +1,156 @@
+{
+    "project_one_line": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/OG-USA",
+        "type": "html",
+        "data": "<p>Dynamic Tax Scoring Model</p>"
+    },
+    "key_features": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "project_overview": {
+        "start_header": "OG-USA",
+        "end_header": "Disclaimer",
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
```

Comment:
Maybe for data field here:
`<p>OG-USA is an overlapping-generations (OG) model of the economy of the United States (USA) that allows for dynamic general equilibrium analysis of federal tax policy. The model output focuses changes in macroeconomic aggregates (GDP, investment, consumption), wages, interest rates, and the stream of tax revenues over time. Careful documentation of the model--its output, and solution method--is available <a href=\"https://github.com/rickecon/OG-USA/blob/doc/documentation/OGUSAdoc.pdf\">here</a> and is regularly updated.</p>`

cc @rickecon 

Also, @rickecon, the above link to the documentation is what we have on the OG-USA GitHub repo, but it links to to a branch on your fork of OG-USA.  Should this be part of the OG-USA master repo?